cylc/uiserver/schema.py

```diff
@@ -170,7 +148,7 @@ class Arguments:
             ''')
         )
         mode = RunMode(
-            default_value=RunMode.Live.name  # type: ignore[attr-defined]
+            default_value=RunMode.Live.name
```

Comment:
Is inheriting `graphene.Enum` a purely convenience thing?

Reply:
It's importing `cylc.flow.network.schema.WorkflowRunMode` ([which is a `graphene.Enum`](https://github.com/cylc/cylc-flow/blob/5b2b1808f1c8f7056efabd2a5a5fe1a93de39690/cylc/flow/network/schema.py#L627-L631)) as `RunMode`